platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationConfigInstaller.java

```diff
@@ -83,58 +82,63 @@ public void run() {
       LOGGER.debug("No config file located, cannot load from it.");
       return;
     }
-    InputStream is = null;
-    try {
-      is = new FileInputStream(configFile);
+
+    try (InputStream is = new FileInputStream(configFile)) {
```

Comment:
❗️ We shouldn't be using try-with-resources on input stream as it will throw an exception if it fails to close the stream at the end but at that point, we already got everything we needed from it. This is why the original code was using `IOUtils.closeQuietly()`.  Although, just putting it in a finaly clause should be good enough.

This is a good candidate for a suppression.